GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -368,53 +368,32 @@ private void FromTextUpdate(object sender, EventArgs e)
 
         private void ToTextUpdate(object sender, EventArgs e)
         {
-            string destinationPath = string.Empty;
+            bool destinationUnfilled = string.IsNullOrEmpty(_NO_TRANSLATE_To.Text);
+            bool subDirectoryUnfilled = string.IsNullOrEmpty(_NO_TRANSLATE_NewDirectory.Text);
 
-            if (string.IsNullOrEmpty(_NO_TRANSLATE_To.Text))
-            {
-                destinationPath += "[" + label2.Text + "]";
-            }
-            else
-            {
-                destinationPath += _NO_TRANSLATE_To.Text.TrimEnd('\\', '/');
-            }
-
-            destinationPath += "\\";
+            string destinationDirectory = destinationUnfilled ? $"[{destinationLabel.Text}]" : _NO_TRANSLATE_To.Text;
+            string destinationSubDirectory = subDirectoryUnfilled ? $"[{subdirectoryLabel.Text}]" : _NO_TRANSLATE_NewDirectory.Text;
 
-            if (string.IsNullOrEmpty(_NO_TRANSLATE_NewDirectory.Text))
-            {
-                destinationPath += "[" + label3.Text + "]";
-            }
-            else
-            {
-                destinationPath += _NO_TRANSLATE_NewDirectory.Text;
-            }
+            string destinationPath = Path.Combine(destinationDirectory, destinationSubDirectory);
 
-            Info.Text = string.Format(_infoNewRepositoryLocation.Text, destinationPath);
+            string newRepositoryLocationInfo = string.Format(_infoNewRepositoryLocation.Text, destinationPath);
 
-            if (destinationPath.Contains("[") || destinationPath.Contains("]"))
+            if (destinationUnfilled || subDirectoryUnfilled)
             {
+                Info.Text = newRepositoryLocationInfo;
                 Info.ForeColor = Color.Red;
                 return;
             }
 
-            if (Directory.Exists(destinationPath))
+            if (Directory.Exists(destinationPath) && Directory.EnumerateFileSystemEntries(destinationPath).Any())
             {
-                if (Directory.GetDirectories(destinationPath).Length > 0 || Directory.GetFiles(destinationPath).Length > 0)
-                {
-                    Info.Text += " " + _infoDirectoryExists.Text;
-                    Info.ForeColor = Color.Red;
-                }
-                else
-                {
-                    Info.ForeColor = SystemColors.ControlText;
-                }
-            }
-            else
-            {
-                Info.Text += " " + _infoDirectoryNew.Text;
-                Info.ForeColor = SystemColors.ControlText;
+                Info.Text = $@"{newRepositoryLocationInfo} {_infoDirectoryExists.Text}";
```

Comment:
Is "$@" really necessary here? If yes, I guess it should also be used above.

Reply:
I am not sure if it should be translated (there's a warning about localizable string if it's not a verbatim string).

Reply:
IIUIC, `$@"..."` avoids the special handling of backslashes `\` in literal(?) strings. I guess this has no effect on replaced variables. Hence `$@"..."` is pointless in this function, i.e. `$"..."` should suffice. @C# experts, am I right?

Reply:
`@` marks the string as "verbatim", e.g. don't parse escaping, `$` is an interpolation.
so together they make a "verbatim interpolated string"

Reply:
I've tested it: `$@{s}` and `${s}` yield the same string also if `s` contains a backslash.
So, since we don't use backslashes, i.e. no potential escaping, in this file, "verbatim" has no effect - neither positive nor negative - and _could_ be omitted.